changelog/fragments/1740762770-fix-race-conditions-in-rate-limit-processor.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
```

Comment:
@mauri870 we don't use the changelog fragments for Beats ;)

You need to add an entry in `CHANGELOG.next.asciidoc` instead.